Readme.md

```diff
@@ -12,13 +12,22 @@ Do you want to quickly try out this project and see it in action? Check it out i
 
 We have a benchmark project to measure the performance of the validation process. You can find it here: [`Lombiq.EInvoiceValidator.Benchmark`](./Lombiq.EInvoiceValidator.Benchmark/Readme.md).
 
+## Projects
+
+- The main library for validating eInvoices - [Lombiq.EInvoiceValidator](./Lombiq.EInvoiceValidator/Lombiq.EInvoiceValidator.csproj)
+- A sample console application demonstrating how to use the library - [Lombiq.EInvoiceValidator.Sample](./Lombiq.EInvoiceValidator.Sample/Readme.md)
+- A benchmark project to measure the performance of the validation process. - [Lombiq.EInvoiceValidator.Benchmark](./Lombiq.EInvoiceValidator.Benchmark/Readme.md)
+- Unit tests for the library. - [Lombiq.EInvoiceValidator.Tests](./Lombiq.EInvoiceValidator.Tests/Lombiq.EInvoiceValidator.Tests.csproj)
+
 ## Prerequisites
```

Comment:
Elaborate whether there's any difference here between development and production.

Reply:
There is no difference.

Reply:
If you think about our very specific case where we need to do special Azure startup commands, I believe that's very specific to us. I think listing Node.js as a prerequisite is enough.